DDDEastAnglia/DataAccess/DefaultSponsorSorter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.DataAccess
+{
+    public static class SponsorExtensions
+    {
+        public static IEnumerable<Sponsor> OrderBySponsorSorter(this IEnumerable<Sponsor> sponsors)
+        {
+            return new DefaultSponsorSorter().Sort(sponsors);
+        }
+    }
+
+    public class DefaultSponsorSorter : ISponsorSorter
```

Comment:
[This was sealed](https://github.com/adrianbanks/dddeastanglia/compare/add-sponsors-to-db...CraftyFella:pull-sponsors-for-sidebar#diff-f010e181923c8492ec8d18b90389e2f1L9), but is no longer. Was that intentional?